Assets/Scripts/Entities/Enemies/Enemy.cs

```diff
@@ -20,6 +20,8 @@ public abstract class Enemy : Entity
     private float currentTargetUpdateCooldown = 0f;
     private float attackCooldown = 0f;
 
+    public static List<Enemy> list = new();
```

Comment:
Remove this list and use the player current room remainingEnemies variable instead to detect when there is no more enemy in the room

Reply:
Change the music the way I wanted will be too long to do because of some trouble there is, can't be done before the review, so we only use the calm music for the menu